core/dbt/artifacts/schemas/run/v5/run.py

```diff
@@ -34,6 +37,8 @@ class RunResult(NodeResult):
     agate_table: Optional["agate.Table"] = field(
         default=None, metadata={"serialize": lambda x: None, "deserialize": lambda x: None}
     )
+    # TODO: Do we need to do the field(...) stuff from above?
```

Comment:
I think we don't need this since we are okay to serialize / deserialize minor schema evolutions.